package.json

```diff
@@ -6,16 +6,20 @@
     "start": "node ./dev-server/index.js",
     "test": "npm run test:lint && npm run build && npm run test:unit",
     "test:lint": "eslint . --ext .js,.jsx,.json",
+    "test:lint:ci": "eslint . --ext .js,.jsx,.json --format junit -o ./test/results/lint-results.xml",
     "test:integration": "npm run test:integration:jest && npm run test:smoke",
     "test:integration:jest": "jest ./test/integration/*.test.js",
     "test:integration:remote": "npm run test:integration:jest:remote && npm run test:smoke:sauce",
-    "test:integration:jest:remote": "SMOKE_REMOTE=true jest ./test/integration/*.test.js",
-    "test:smoke": "tap ./test/integration-legacy/smoke-testing/*.js --timeout=3600 --no-coverage -R classic",
+    "test:integration:jest:remote": "SMOKE_REMOTE=true JEST_JUNIT_OUTPUT_NAME=integration-jest-results.xml jest ./test/integration/*.test.js",
+    "test:smoke": "tap ./test/integration-legacy/smoke-testing/*.js --timeout=3600 --no-coverage -R classic --output-file ./test/results/integration-raw-tap.tap",
```

Comment:
Will these changes make it harder for us to run these things locally? If we can configure this in the CI config, I think that would be better, and leave the configuration here as we would want it for local testing.

Reply:
They shouldn't.  They both output normally to the console.  You may need to create an /integration directory in ./test.  I have tested it locally, and both of these tests output useful to the terminal.

I will have to make a separate PR to update the version of Chromedriver, which I needed to do to run locally, but that is independent of the changes in this PR.

Reply:
I've updated the readme to include information about needing a ./test/results directory.

Reply:
If `test/results` will get created when you run tests, then we should probably ignore it in `.gitignore` with this PR. Otherwise it will start showing up in people's working copies and then it's just a matter of time before it gets committed, which we don't want.

Reply:
It's already in there